src/main/java/pb360/json/JacksonObjectMapper.java

```diff
@@ -0,0 +1,35 @@
+package pb360.json;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import com.fasterxml.jackson.core.JsonGenerationException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import pb360.model.RestAPI;
+
+public class JacksonObjectMapper {
+	public static void main(String[] args) throws IOException{
+
+		try {
+		ObjectMapper objectMapper = new ObjectMapper();
+		RestAPI restApi =objectMapper.readValue(new File("D:\\restapi\\JSON_Sample.json"), RestAPI.class);
```

Comment:
help me explain about the function of this class, maybe it is not using on the project

Reply:
i try to use this to read json file